nilearn/decomposition/multi_pca.py

```diff
@@ -147,12 +81,16 @@ class MultiPCA(BaseEstimator, TransformerMixin, CacheMixin):
         The number of CPUs to use to do the computation. -1 means
         'all CPUs', -2 'all CPUs but one', and so on.
 
+    max_nbytes: int,
+        Size (in bytes) above which the intermediary unmasked data will be
```

Comment:
could still be clearer...the difference between "in memory" and memmap file "on disk" could be more explicit. As well as what the decision rule is exactly.